### PR TITLE
Add POST /announcement test scenario

### PR DESCRIPTION
**Pull request by dannyh79**
#5

This adds POST /announcement scenario with fixed API response. Actual impl to be followed up.

### CODE REVIEW

File: features/announcement.feature [hunk 1]
```diff
@@ -84,3 +84,14 @@ Feature: Announcement
         }
       ]
       """
+  Scenario: POST /announcement
+    When I make a POST request to "/announcement"
+      | msg_en        | msg_zh   | uri                                           | role         |
+      | hello world   | 世界你好 | https://testability.opass.app/announcements/1 | ["audience"] |
```
Comment (dannyh79) [inline features/announcement.feature:90]: Ref: https://github.com/CCIP-App/CCIP-Server/blob/master/app/ccip.py#L309-L312

File: features/announcement.feature [hunk 2]
```diff
@@ -84,3 +84,14 @@ Feature: Announcement
         }
       ]
       """
+  Scenario: POST /announcement
+    When I make a POST request to "/announcement"
```
Comment (elct9620) [inline features/announcement.feature]: 這邊應該用 Doc String 放整段 JSON 比較符合實際使用

**Review by elct9620 — CHANGES_REQUESTED** [review]

File: features/announcement.feature (hunk 2 above)
Comment (dannyh79) [reply features/announcement.feature]: 感謝建議 :D

Comment (dannyh79) [reply features/announcement.feature]: ecaca90

**Review by elct9620 — APPROVED** [review]

**Pull request merged**